csharp-advanced-topics/DeprecationInCsharp/DeprecationTests/DeprecationTest.cs

```diff
@@ -1,4 +1,4 @@
-
+using Deprecation.StringUtils;
```

Comment:
You only need this in one place. Either here or in global usings. Since this is the namespace under test in this testing assembly it makes sense for it to live in the global usings. This simplifies the file. Since you have now added it to the global using space it is redundant to have it here. If you are using VS it actually shows that the using here is not needed because it is now in the global usings.

![image](https://github.com/CodeMazeBlog/CodeMazeGuides/assets/1489638/6c205a09-9ad9-49a8-9fb3-2098b94c2fda)